tests/greenboot-bootc-anaconda-iso.sh

```diff
@@ -38,6 +38,7 @@ case "${ID}-${VERSION_ID}" in
         BIB_URL="quay.io/centos-bootc/bootc-image-builder:latest"
         BOOT_ARGS="uefi"
         sudo dnf install -y rpmbuild rust-packaging
+        sudo dnf install -y 'crate(anyhow/default)' 'crate(clap/default)' 'crate(clap/derive)' 'crate(config/default)' 'crate(env_logger/default)' 'crate(glob/default)' 'crate(once_cell/default)' 'crate(pretty_env_logger/default)' 'crate(serde_json/default)' 'crate(tempfile/default)' 'crate(thiserror/default)' 'crate(config/default)' 'crate(env_logger/default)' 'crate(nix/default)'
```

Comment:
@yih-redhat  These crate style install  will not work for  centOS, you can install the packages mentioned in spec file (BuildRequires).

Reply:
ok, I will change to install the packages.